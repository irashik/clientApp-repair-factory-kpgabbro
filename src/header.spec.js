import React from 'react';

import HeaderComponent  from './header';



       jest.mock('react-router-bootstrap');
       jest.mock('react-bootstrap/Button');
       jest.mock('react-bootstrap/ButtonGroup');
      // jest.mock('react-bootstrap/Nav');
      // jest.mock('react-bootstrap/Navbar');



describe("<HeaderComponent />", () => {
    test('should render correctly', () => {

        const component = shallow(<HeaderComponent />)
        expect(component).toMatchSnapshot();
    });
});



