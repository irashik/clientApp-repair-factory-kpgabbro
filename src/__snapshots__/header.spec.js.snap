// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderComponent /> should render correctly 1`] = `
<header>
  <Container
    fluid={true}
  >
    <h1>
      Журнал ремонтов
    </h1>
    <h3>
      АО "КП-Габбро", Карьер Новый Поселок
    </h3>
  </Container>
</header>
`;
